web/locales/en.yml

```diff
@@ -24,9 +24,13 @@ en: # <---- change this to your locale code
   ShowAll: Show All
   CurrentMessagesInQueue: Current jobs in <span class='title'>%{queue}</span>
   Delete: Delete
+  Pause: Pause
```

Comment:
Please put new strings at the bottom of the file so other languages can see an easy diff of which strings are missing.